stepup/demogssp/docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  demogssp:
+    image: ghcr.io/openconext/openconext-basecontainers/${DEMOGSSP_PHP_IMAGE:-php72-apache2-node14-composer2:latest}
+    volumes:
+      - ${DEMOGSSP_CODE_PATH}:/var/www/html
```

Comment:
Thanks for adding this override. Had that one as an uncommited change on my local machine.

One consideration. The mother docker-compose still sets the APP_ENV=prod for this service. When you start the dev override. The env var is not overridden.

Please remove the APP_ENV reference(s) from the base `/stepup/docker-compose.yml`. I see that was also the case for other apps.